landlab/components/stream_power/cfuncs.pyx

```diff
@@ -4,6 +4,9 @@ cimport cython
 
 #from libc.math cimport fabs
 
+# as best as KRB can tell, the function _brentq is in c.
+# not sure what, if anything I need to do to 
+from scipy.optimize._zeros import _brentq as brentq
 
```

Comment:
The part of the code that I would point out as potentially needing the most improvement is making sure that the code used for the brentq root finding is fully cythonized. 

As best as I can tell, this function, spicy.optimize._zeros._brentq is in C and is what is actually run when one uses
scipy.optimize.brentq.

However there may be a better way to import it into this cfuncs file. 

Most of what I know about cython I learned this morning... so if @SiccarPoint @mcflugen or @gregtucker have recommendations, please commit.